mars/utils.py

```diff
@@ -91,6 +91,7 @@
 reset_id_random_seed = reset_id_random_seed
 new_random_id = new_random_id
 _create_task = asyncio.create_task
+_is_ci = (os.environ.get("CI") or "0").lower() in ("1", "true")
```

Comment:
There is `_is_ci` in `mars/serialization/serializables/core.py` and `mars/serialization/serializables/field.py`. Maybe we can use this to instead them.

Reply:
Thanks. I will replace the `_is_ci` in  serializables by this one.